src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -1,6 +1,6 @@
 package org.jlleitschuh.gradle.ktlint
 
-import com.android.build.gradle.BaseExtension
+import com.android.build.gradle.*
```

Comment:
Don't use star imports please

Reply:
Ktlint-gradle should run ktlint on itself 😄

Reply:
Fixed in 7ddc8b03b562446d64edebf1b4d7eec3f0bdbb09